doc/content/gateways/lora-basics-station/troubleshooting.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "Troubleshooting LBS"
+description: ""
+---
+
+This section contains help for common problems you may encounter when using {{% lbs %}} to connect to {{% tts %}}.
+
+<!--more-->
+
+## What is my server address?
+
+This is the address you use to access {{% tts %}}. If you followed the [Getting Started guide]({{< ref "/getting-started" >}}) this is the same as what you use instead of `thethings.example.com`.
+
+CUPS uses the URI: `https://<server-address>:443`
+
+LNS uses the URI: `wss://<server-address>:8887`
+
+## How do I find the CA Trust?
+
+Some device manufacturers include [common CA's](https://www.ccadb.org/) in device firmware, and these devices should connect automatically to {{% tts %}}.
+
+If your device does not contain common CA's, and you are using Let's Encrypt to secure your domain, you may download the LEX3 Trust file [here](https://letsencrypt.org/certs/lets-encrypt-x3-cross-signed.pem.txt). Save the contents of the file as `cert.pem` and upload it as the Server Certificate on your gateway when connecting to {{% lbs %}}.
```

Comment:
`LEX3` is not really a common way of referring to it. `Lets Encrypt DST X3` is more common.